addon/services/scheduler.js

```diff
@@ -195,6 +203,14 @@ if (DEBUG) {
       return hasActiveQueue && hasTasks;
```

Comment:
I believe this won't do what you want. 

```
false && false === false
```

```
false && true === false
```

I believe you want

```
false || true === false
```

we should be using an `||` here, not an `&&`.

Reply:
I think this still holds true. You don't want to falsely (pun intended) report that it doesn't have active queues _and_ active tasks. I think using an && will result in you reporting `hasActiveQueue() === false` when `hasActiveQueue === false` && `hasTasks === true`. Am I wrong here?

Reply:
I looked over the logic and I agree. I updated to use `||`